src/nunit.analyzers.tests/ConstActualValueUsage/ConstActualValueUsageCodeFixTests.cs

```diff
@@ -45,6 +45,27 @@ public void Test()
                 fixTitle: ConstActualValueUsageCodeFix.SwapArgumentsDescription);
         }
 
+        [TestCase(nameof(ClassicAssert.AreEqual))]
+        public void LiteralArgumentIsProvidedForClassicAssertWithDeltaCodeFix(string classicAssertMethod)
+        {
+            var code = TestUtility.WrapMethodInClassNamespaceAndAddUsings($@"
+                public void Test()
+                {{
+                    int expected = 5;
+                    ClassicAssert.{classicAssertMethod}(expected, ↓1, 1);
```

Comment:
It is never good to use two arguments with the same value as it doesn't proof that the correct argument is exchanged.
I have update the code to use `double.Epsilon` for the accuracy.